src/app.js

```diff
@@ -0,0 +1,106 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+
+const { Step, Token } = require("./gherkin/index.js");
+const { suite } = require("./suite/index.js");
+
+// Register new contexts here
+require("./setup.js");
+require("./stepDefinitions/filesContext.js");
+require("./stepDefinitions/generalContext.js");
+require("./stepDefinitions/notificationsContext.js");
+require("./stepDefinitions/provisioningContext.js");
+require("./stepDefinitions/publicLinkContext.js");
+require("./stepDefinitions/sharingContext.js");
+require("./stepDefinitions/webdavContext.js");
+
+// Create Express Server
+const app = express();
+
+app.use(bodyParser.json());
+
+// Configuration
+const PORT = 3000;
+const HOST = "localhost";
+
+app.use("/execute", async (req, res) => {
+  if (req.method !== "POST") {
+    res.writeHead(405).end();
+  }
+  let { pattern, table, token } = req.body;
```

Comment:
I fell it would be more readable when using the API to have the token as part of the step string itself, just as it comes from the feature file
```suggestion
  let { step, table } = req.body;
```

Maybe we can optionally allow to have the keyword be in the step